### PR TITLE
Updates to ingress-perf infra nodes

### PR DESCRIPTION
**Pull request by krishvoor**
This PR updates the Infra nodes instance types,
and updating them to `r5.xlarge`

### CODE REVIEW

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-data-path-9nodes-periodic

**Comment by krishvoor** [comment]
/pj-rehearse periodic-ci-openshift-qe-ocp-qe-perfscale-ci-main-aws-4.14-ocp-qe-perfscale-aws-data-path-9nodes-periodic

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml [hunk 1]
```diff
@@ -86,6 +86,8 @@ tests:
       COMPUTE_NODE_REPLICAS: "9"
       COMPUTE_NODE_TYPE: m5.xlarge
       LOKI_USE_SERVICEMONITOR: "false"
+      OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
```
Comment (jtaleric) [inline ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:89]: The commit message mentions a different instance size? 
```This PR updates the Infra nodes instance types, and updating them to c5.4xlarge```

**Review by jtaleric — CHANGES_REQUESTED** [review]
What size do we want?

File: ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml (hunk 1 above)
Comment (krishvoor) [reply ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.14.yaml:89]: Based on the Slack Conversation from [yesterday](https://redhat-internal.slack.com/archives/C05CDC19ZKJ/p1694530732701539), Raul will help "fit" the ingress benchmark configuration for `r5.xlarge` which is the default node-type of infra. nodes in ROSA/ Managed Service

> `This PR updates the Infra nodes instance types, and updating them to c5.4xlarge`

will update the commit msg.

**Comment by jtaleric** [comment]
Can we hold this PR until Raul has the PR ready for ingress-perf?

**Comment by krishvoor** [comment]
Closing in favor of this: https://github.com/openshift/release/commit/1daab6f5dcb173cb1f3e88fb3b2a110cef3f9800

**Pull request closed without merging**